Stripe/STPAddCardViewController.m

```diff
@@ -328,9 +330,40 @@ - (void)nextPressed:(__unused id)sender {
             } else {
                 NSString *phone = self.rememberMePhoneCell.contents;
                 NSString *email = self.emailCell.contents;
-                BOOL rememberMeSelected = [STPEmailAddressValidator stringIsValidEmailAddress:email] && [STPPhoneNumberValidator stringIsValidPhoneNumber:phone] && self.showingRememberMePhoneAndTerms;
+
+                /**
+                 Remember me button usage segmented into following categories:
+                 1. User saw the toggle, selected it, and entered valid email/phone details
+                 2. User saw the toggle, and did not select it or entered invalid details
+                 3. User did not see the toggle because the developer disabled it
+                 4. User did not see the toggle because the developer enabled it but they were otherwise ineligible
+                    (eg payment context flow disables if you already have >0 payment methods,
+                     or you had a cancelled SMS fill)
+                 5. User saw the email field and added card by SMS code from a previous Remember Me save
+                 */
+
+                STPAddCardRememberMeUsage rememberMeSelected;
```

Comment:
nit: might be less confusing to call this `rememberMeUsage`, since it's no longer a bool